### ISSUE TITLE
[BUG] Conflict with Chipped by Terrarium and Diversity Version 2.0.0

### ISSUE THREAD

**Issue by Kw126** [FaeWulf/Diversity#16]
### Current Behavior

game crashes when Diversity 2.0.0 and Chipped are in a mod pack together in all versions of Minecraft currently supported by Diversity.
Diversity version 1.3.4 does not crash with Chipped in 1.21 or 1.21.1.

### Expected Behavior

the game to run

### Steps To Reproduce

I'm using the newest version of modrinth as a launcher
Here is a zip of the MRPACK File of a mod pack in 1.20.1 containing only relevant mods
[Diversity and Chipped conflict 1.0.0.zip](https://github.com/user-attachments/files/16925119/Diversity.and.Chipped.conflict.1.0.0.zip)
if that isn't working here is a google drive with a unzipped version of the pack and a folder containing the mods individually
https://drive.google.com/drive/folders/1CdJBwkRdEt2EcdX9Z257jRirFn9Rq0w9?usp=drive_link

### Additional info

```markdown
- **Game version**: 1.20.1 (Reminder that it Occurs in 1.21 and 1.21.1 too)
- **Mod version**: 2.0.0
- **Mod list**:
Athena by Terrarium                  V 3.1.2
Chipped by Terrarium                 V 3.0.6
Diversity by FaeWulf                 V 2.0.0
Fabric API by modmuss50              V 0.92.2+1.20.1
Resourceful Lib by Team Resourceful  V 2.1.29
```

### Crash log/ Gamelog

Crash Log: https://mclo.gs/sj4eRep

**Comment by Asylum-Media** [comment]
I _think_ I have a similar issue.
While reference is made to Chipped, we don't have the mod in our pack.
The issue is the same, the game will not run with 2.0.0 of Diversity, but will with 1.3.4.
https://mclo.gs/sI7QekS

**Comment by FaeWulf** [comment]
> I _think_ I have a similar issue. While reference is made to Chipped, we don't have the mod in our pack. The issue is the same, the game will not run with 2.0.0 of Diversity, but will with 1.3.4. https://mclo.gs/sI7QekS

```
Java.lang.IllegalStateException: Invalid block entity handcrafted:oven // earth.terrarium.handcrafted.common.blockentities.OvenBlockEntity state at class_2338{x=-100, y=63, z=-196}, got Block{handcrafted:oven}[facing=north,lit=false]
```

Looks like it is an issue of `Handcrafted`. And I don't see `Diversity` being use in the crash log.

**Comment by FaeWulf** [comment]
Fixed in version 2.0.1, check it out! 
https://modrinth.com/mod/faewufs-diversity/versions
